src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -783,6 +783,15 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 				case IntrinsicId.Type_GetType: {
 						reflectionContext.AnalyzingPattern ();
 
+						var parameters = calledMethod.Parameters;
+						if ((parameters.Count == 3 && (methodParams[2].AsConstInt () == null || methodParams[2].AsConstInt () != 0)) ||
```

Comment:
There are two overloads of `Type.GetType` with 3 parameters so this needs to be narrowed down.